config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -3180,19 +3180,24 @@ presubmits:
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240705-131cd74733-master
+        # This picks the latest available k8s.io/api/resource version that is available.
+        # May be different depending on the PR.
         command:
-        - runner.sh
-        - /workspace/scenarios/kubernetes_e2e.py
+        - /bin/sh
         args:
-        - --deployment=node
-        - --env=KUBE_SSH_USER=core
-        - --gcp-zone=us-west1-b
-        - '--node-test-args=--feature-gates="DynamicResourceAllocation=true" --service-feature-gates="DynamicResourceAllocation=true" --runtime-config=resource.k8s.io/v1alpha2=true --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
-        - --node-tests=true
-        - --provider=gce
-        - '--test_args=--label-filter="Feature: containsAny DynamicResourceAllocation && !Flaky"'
-        - --timeout=65m
-        - --node-args=--image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv1-serial.yaml
+        - -c
+        - >-
+          runner.sh
+          /workspace/scenarios/kubernetes_e2e.py
+          --deployment=node
+          --env=KUBE_SSH_USER=core
+          --gcp-zone=us-west1-b
+          --node-test-args="--feature-gates=DynamicResourceAllocation=true --service-feature-gates=DynamicResourceAllocation=true --runtime-config=resource.k8s.io/$(echo pkg/apis/resource/v1* | tail -1 | xargs basename)=true --container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags=\"--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service\" --extra-log=\"{'name': 'crio.log', 'journalctl': '-u', 'crio']}\""
```

Comment:
The nesting of command line args which need to be quoted correctly here is insane. I *think* `--node-test-args` takes a string which then contains `--kubelet-flags` and `--extra-log` which both again get strings that contain spaces.

I hope I got this right when converting to a single YAML string that gets passed to /bin/sh, but I am not sure. Let's try it...

/assign @dims